### PR TITLE
Add the Houses and Lots pages

### PR DESCRIPTION
**Pull request by ricardoruwer**
☝️

### CODE REVIEW

File: test/construtora_lc_hiert_web/controllers/lot_controller_test.exs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+defmodule ConstrutoraLcHiertWeb.LotControllerTest do
+  use ConstrutoraLcHiertWeb.ConnCase
+
+  describe "GET /imoveis/lotes" do
+    test "accesses the lots page", %{conn: conn} do
+      conn = get(conn, "/imoveis/lotes")
+
+      assert html_response(conn, 200)
```
Comment (w-sanches) [inline test/construtora_lc_hiert_web/controllers/lot_controller_test.exs]: If I am not wrong, this is a 200 response, but with the "not found" page, right?

Do you think these tests should check if the property loads correctly when there are lots (or houses as this comment also applies to the house controller test.

Comment (ricardoruwer) [reply test/construtora_lc_hiert_web/controllers/lot_controller_test.exs]: Like that? 9654f676e4776b1ae58cadc8b0c8bdd0261876f5

File: test/construtora_lc_hiert_web/controllers/apartment_controller_test.exs [hunk 2]
```diff
@@ -1,11 +1,43 @@
 defmodule ConstrutoraLcHiertWeb.ApartmentControllerTest do
   use ConstrutoraLcHiertWeb.ConnCase
 
+  alias ConstrutoraLcHiert.Properties
+
   describe "GET /imoveis/apartamentos" do
-    test "accesses the apartments page", %{conn: conn} do
+    test "accesses the page when there is apartments", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/apartment_controller_test.exs]:
```suggestion
    test "accesses the page when there are apartments", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/apartment_controller_test.exs [hunk 3]
```diff
@@ -1,11 +1,43 @@
...
+      property = property_fixture()
+
       conn = get(conn, "/imoveis/apartamentos")
 
-      assert html_response(conn, 200)
+      assert html_response(conn, 200) =~ property.address
     end
+
+    test "show no results found when there is no apartments", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/apartment_controller_test.exs]:
```suggestion
    test "show no results found when there are no apartments", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/house_controller_test.exs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+defmodule ConstrutoraLcHiertWeb.HouseControllerTest do
+  use ConstrutoraLcHiertWeb.ConnCase
+
+  alias ConstrutoraLcHiert.Properties
+
+  describe "GET /imoveis/casas" do
+    test "accesses the page when there is houses", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/house_controller_test.exs]:
```suggestion
    test "accesses the page when there are houses", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/house_controller_test.exs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      property = property_fixture()
+
+      conn = get(conn, "/imoveis/casas")
+
+      assert html_response(conn, 200) =~ property.address
+    end
+
+    test "show no results found when there is no houses", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/house_controller_test.exs]:
```suggestion
    test "show no results found when there are no houses", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/lot_controller_test.exs [hunk 6]
```diff
@@ -0,0 +1,43 @@
+defmodule ConstrutoraLcHiertWeb.LotControllerTest do
+  use ConstrutoraLcHiertWeb.ConnCase
+
+  alias ConstrutoraLcHiert.Properties
+
+  describe "GET /imoveis/lotes" do
+    test "accesses the page when there is lots", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/lot_controller_test.exs]:
```suggestion
    test "accesses the page when there are lots", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/lot_controller_test.exs [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      property = property_fixture()
+
+      conn = get(conn, "/imoveis/lotes")
+
+      assert html_response(conn, 200) =~ property.address
+    end
+
+    test "show no results found when there is no lots", %{conn: conn} do
```
Comment (Lucasnar) [inline test/construtora_lc_hiert_web/controllers/lot_controller_test.exs]:
```suggestion
    test "show no results found when there are no lots", %{conn: conn} do
```

File: test/construtora_lc_hiert_web/controllers/lot_controller_test.exs (hunk 1 above)
Comment (w-sanches) [reply test/construtora_lc_hiert_web/controllers/lot_controller_test.exs]: Yeah, LGTM

**Pull request merged**